test/builder/graphql/issue.js

```diff
@@ -18,7 +18,7 @@ export const IssueBuilder = createSpecBuilderClass('Issue', {
   number: {default: 123},
   state: {default: 'OPEN'},
   bodyHTML: {default: '<h1>HI</h1>'},
-  author: {linked: UserBuilder},
+  author: {linked: UserBuilder, default: null},
```

Comment:
I really like defaulting these to `null` in the builder. This way we'll catch _anywhere_ that an author is fetched from GraphQL but we don't account for it being nullable 👍